.github/workflows/release.yml

```diff
@@ -0,0 +1,42 @@
+# See: https://github.com/bitcoin-core/secp256k1/blob/master/doc/release-process.md.
```

Comment:
```suggestion
# See: https://github.com/bitcoin-core/secp256k1/blob/master/doc/release-process.md
```
nit: link is broken otherwise

Reply:
Thanks! [Fixed](https://github.com/bitcoin-core/secp256k1/pull/1413#issuecomment-1697108649).